study-group/Toast-Implementation-Studies.md

```diff
@@ -90,9 +90,151 @@ The `restoreFocusOnClear` attribute restores page's focus once the toast is clea
 
 *TODO: Fill in study for this library*
 
-## ngx-toastr
+## [ngx-toastr](https://github.com/scttcper/ngx-toastr)
+ngx-toastr is a library from author [@scttcper](https://github.com/scttcper) to create a toast notification built for Angular.
 
-*TODO: Fill in study for this library*
+
+### Design & Sample Code
+Before using ngx-toastr, setup is required. First, add the CSS:
+
+```js
+// regular style toast
+@import '~ngx-toastr/toastr';
+
+// bootstrap style toast
+@import '~ngx-toastr/toastr-bs4-alert';
+```
+
+Next, add ToastrModule and BrowserAnimationsModule to app NgModule:
+
+```js
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { ToastrModule } from 'ngx-toastr';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    BrowserAnimationsModule, // required animations module
+    ToastrModule.forRoot() // ToastrModule added
+  ],
+  ...
+})
+...
+```
+
+Once the modules are set up, this code will display a toast:
+
+```js
```

Comment:
ts, not js, I think

Reply:
Sometimes I forget ts exists . . . fixing that now